docs/content/docs/connectors/datastream/pulsar.md

```diff
@@ -169,13 +224,32 @@ you can use the predefined `PulsarDeserializationSchema`. Pulsar connector provi
   PulsarDeserializationSchema.pulsarSchema(Schema, Class, Class);
   ```
 - Decode the message by using Flink's `DeserializationSchema`
+  {{< tabs "pulsar-deserializer-deserialization-schema" >}}
+  {{< tab "Java" >}}
   ```java
   PulsarDeserializationSchema.flinkSchema(DeserializationSchema);
   ```
+  {{< /tab >}}
+  {{< tab "Python" >}}
+  ```python
+  PulsarDeserializationSchema.flink_schema(DeserializationSchema)
+  ```
+  {{< /tab >}}
+  {{< /tabs >}}
+
 - Decode the message by using Flink's `TypeInformation`
+  {{< tabs "pulsar-deserializer-type-information" >}}
+  {{< tab "Java" >}}
   ```java
   PulsarDeserializationSchema.flinkTypeInfo(TypeInformation, ExecutionConfig);
   ```
+  {{< /tab >}}
+  {{< tab "Python" >}}
+  ```python
+  PulsarDeserializationSchema.flink_type_info(TypeInformation, ExecutionConfig)
```

Comment:
Currently, it's not possible to create ExecutionConfig in Python and so I have merged one commit(https://github.com/apache/flink/commit/73590f432d8dc4cd53f0357025e250a823e6b857) to allow  create PulsarDeserializationSchema without specifying ExecutionConfig. So what about suggest users to use `PulsarDeserializationSchema.flink_type_info(TypeInformation)` here?